### PR TITLE
Automate dependency installation

### PR DESCRIPTION
**Pull request by VasiliiGeneralov**
Dependency installation for following platforms is automated in this commit:
  * Arch-like Linux
  * Debian-like Linux

### CODE REVIEW

**Comment by VasiliiGeneralov** [comment]
@goncharovnikita, could you please submit a patch for macOS?

File: scripts/_prepare_deps/_deps.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+debian_deps: Final[List[str]] = [
+    'libx11-xcb-dev',
+    'libglu1-mesa-dev',
+    'libxrender-dev',
+    'libxi-dev',
+    'libxkbcommon-x11-dev',
+    'libwayland-dev',
+    'libxkbcommon-dev=1.5.0-1',
+]
```
Comment (VasiliiGeneralov) [inline scripts/_prepare_deps/_deps.py]: python-distutils should also be installed, also plain arch installation may lack some other packages like fontconfig2 or so, need to check that

Comment (VasiliiGeneralov) [reply scripts/_prepare_deps/_deps.py]: + autoconf

Comment (VasiliiGeneralov) [reply scripts/_prepare_deps/_deps.py]: + autoconf-archive

Comment (VasiliiGeneralov) [reply scripts/_prepare_deps/_deps.py]: + libtool

Comment (VasiliiGeneralov) [reply scripts/_prepare_deps/_deps.py]: btw, think twice before building on arch, your life will get miserable

Comment (VasiliiGeneralov) [reply scripts/_prepare_deps/_deps.py]: + libxcb*-dev

Comment (VasiliiGeneralov) [reply scripts/_prepare_deps/_deps.py]: just don't build qt from source, do your mental health a favor

File: scripts/prepare_deps.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+from _prepare_deps._build_system import build_system
+from _prepare_deps._system import System
+import subprocess
+import sys
+
+
+def _promt_user():
```
Comment (goncharovnikita) [inline scripts/prepare_deps.py]: typo?

Comment (goncharovnikita) [inline scripts/prepare_deps.py]:
```suggestion
def _prompt_user():
```

**Review by goncharovnikita — APPROVED** [review]

**Pull request merged**